dev/ant_build.js/artifactGAVToFilePath.json

```diff
@@ -0,0 +1,54 @@
+{
+  "com.fasterxml.jackson.core:jackson-annotations:2.2.3": "lib/buildtasks/jackson-annotations-2.2.3.jar",
```

Comment:
I'd rather not be using so many hard-coded versions in here. Where possible, I would like Open Liberty to use just one version of each dependency. It should be easy for us to a) know what our dependencies are and b) update that version in a single place to pull in fixes from upstream.

When I looked in the oss_dependencies.maven file, I was disappointed to find that we have multiple versions of jackson-annotations already.
https://github.com/OpenLiberty/open-liberty/blob/544f05a377f6b5df5b5ead966dd2fe77ced0396f/dev/cnf/oss_dependencies.maven#L11

Reply:
after talking to Katie, we will address this in #15902